src/eeprom.h

```diff
@@ -0,0 +1,323 @@
+/*
+ *  Open HR20
+ *
+ *  target:     ATmega169 @ 4 MHz in Honnywell Rondostat HR20E
+ *
+ *  compiler:   WinAVR-20071221
+ *              avr-libc 1.6.0
+ *              GCC 4.2.2
+ *
+ *  copyright:  2008 Jiri Dobry (jdobry-at-centrum-dot-cz)
+ *
+ *  license:    This program is free software; you can redistribute it and/or
+ *              modify it under the terms of the GNU Library General Public
+ *              License as published by the Free Software Foundation; either
+ *              version 2 of the License, or (at your option) any later version.
+ *
+ *              This program is distributed in the hope that it will be useful,
+ *              but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *              MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *              GNU General Public License for more details.
+ *
+ *              You should have received a copy of the GNU General Public License
+ *              along with this program. If not, see http:*www.gnu.org/licenses
+ */
+
+/*!
+ * \file       eeprom.h
+ * \brief      Keyboard driver header
+ * \author     Jiri Dobry <jdobry-at-centrum-dot-cz>
+ * \date       $Date$
+ * $Rev$
+ */
+
+#pragma once
+
+#include "config.h"
+#include <avr/eeprom.h>
+#include "debug.h"
+#include "main.h"
+#include "common/rtc.h"
+#include "adc.h"
+#if (RFM == 1)
+#include "rfm_config.h"
+#include "common/rfm.h"
+#endif
+
+
+#define EEPROM __attribute__((section(".eeprom")))
+
+typedef struct                                                  // each variables must be uint8_t or int8_t without exception
+{ /* 00 */ uint8_t lcd_contrast;

```

Comment:
If it's easy, it would be great to have a newline and alignment here, otherwise no big deal.